src/d3a/models/myco_matcher/pay_as_bid.py

```diff
@@ -0,0 +1,37 @@
+from d3a.constants import FLOATING_POINT_TOLERANCE
+from d3a.models.market.market_structures import BidOfferMatch
+from d3a.models.myco_matcher.base_matcher import BaseMatcher
+
+
+class PayAsBidMatch(BaseMatcher):
```

Comment:
Also here, I would propose:
```suggestion
class PayAsBidMatcher(BaseMatcher):
```